### ISSUE TITLE
macOS build failed in parsing gcc-lt-34 configuration flag

### ISSUE THREAD

**Issue by izgzhen** [snowleopard/hadrian#449]
```
Error when running Shake build system:
* _build/stage1/rts/libHSrts_thr.a
* _build/stage1/rts/c/Adjustor.thr_o
Configuration flag 'gcc-lt-34 = @GccLT34@' cannot be parsed.
CallStack (from HasCallStack):
  error, called at src/Oracles/Flag.hs:44:48 in main:Oracles.Flag
```

**Comment by bgamari** [comment]
This is almost certainly fallout from https://phabricator.haskell.org/D4165

**Comment by izgzhen** [comment]
I see. I'll give a fix today

**Comment by snowleopard** [comment]
Poor `GccLt34`. Its life was so short (was added to Hadrian two days ago).

**Comment by snowleopard** [comment]
This was fixed in #450.
